### DISCUSSION TITLE
Why is PrimeTime not part of TCA ?

### DISCUSSION THREAD

**Discussion by unknown-address** [pointfreeco/swift-composable-architecture#355] [Q&A] [answered]
Hi there, 

I was wondering why the PrimeTime project, which has been covered for so many episodes in pointfree, is not part of TCA. How come ?  Is there a particular reason ? 

I know and I've seen that the TCA has many examples that are similar to the PrimeTime project.
I think it would be cool to add it anyway.

**Comment by mbrandonw** [comment, marked as answer]
Hey @girardimatthew, I don't think we made a conscious decision to not include it. When deciding which case studies to include we focused mostly on core use cases that many people encounter while building real world apps. In retrospect, I'm not sure PrimeTime fits that bill. It was great as a demo application to introduce the concepts of TCA in our episodes, but I think if a random person stumbled into this repo with no knowledge of Point-Free they may be a little confused by PrimeTime 😄

**Reply by unknown-address** [reply]
ok, sounds good. 
Thanks for the reply.
